src/Pages/Donation/index.tsx

```diff
@@ -24,18 +21,44 @@ const DonationPage: React.FC = () => {
     setFormData({ ...formData, [name]: value });
   };
 
+  const handleDonateButtonClick = () => {
+    // Generate the payment form URL using the Cashfree API
+    const paymentFormUrl =
+      "https://payments-test.cashfree.com/forms/Iskcondonation" +
+      `?appId=${APP_ID}&orderId=${ORDER_ID}&orderAmount=${formData.amount}&orderCurrency=INR&customerName=${formData.name}&customerEmail=${formData.email}&customerPhone=9999999999`;
+
+    // Redirect the user to the payment form URL
+    window.location.href = paymentFormUrl;
```

Comment:
this is how you navigate to another page? Use navigate hooks from the react-router library.